src/App.vue

```diff
@@ -23,7 +25,7 @@ export default {
 
 <style scoped>
 .page-content {
-  min-height: calc(100vh - 64px - 80px);
+  min-height: calc(100vh - 64px - 81px);
```

Comment:
I assume 64 and 81px are the navbar and footer heights. 
This is not so obvious and when reviewing after some time it will be even less obvious.

Therefore would recommend extracting to css variables such things.
Define in a top level css files
```
--navbarHeight: 64px;
--footerHeight: 81px;
```

For navbar styles
```
height: var(--navbarHeight);
```
This specific line.
```css
min-height: calc(100vh - var(--navbarHeight) - var(--footerHeight));
```